code/modules/mob/living/silicon/robot/robot_modules/syndicate.dm

```diff
@@ -43,3 +44,142 @@
 	src.modules -= id
 	id = null
 	return ..()
+
+// Gets a big shield and a gun that shoots really fast to scare the opposing force.
+/obj/item/weapon/robot_module/robot/syndicate/protector
+	name = "protector robot module"
+	sprites = list(
+		"Cerberus - Treaded" = "syndie_treadhound",
+		"Cerberus" = "syndie_bloodhound",
+		"Ares" = "squats",
+		"XI-ALP" = "syndi-heavy"
+		)
+
+/obj/item/weapon/robot_module/robot/syndicate/protector/New(var/mob/living/silicon/robot/R)
+	..()
+	src.modules += new /obj/item/shield_projector/rectangle/weak(src)
+	src.modules += new /obj/item/weapon/gun/energy/dakkalaser(src)
+	src.modules += new /obj/item/weapon/handcuffs/cyborg(src)
+	src.modules += new /obj/item/weapon/melee/baton/robot(src)
+
+// 95% engi-borg and 15% roboticist.
+/obj/item/weapon/robot_module/robot/syndicate/mechanist
+	name = "mechanist robot module"
+	sprites = list(
+		"XI-GUS" = "spidersyndi",
+		"WTOperator" = "sleekhos"
+		)
+
+/obj/item/weapon/robot_module/robot/syndicate/mechanist/New(var/mob/living/silicon/robot/R)
+	..()
+	// General engineering/hacking.
+	src.modules += new /obj/item/borg/sight/meson(src)
+	src.modules += new /obj/item/weapon/weldingtool/electric/mounted/cyborg(src)
+	src.modules += new /obj/item/weapon/screwdriver/cyborg(src)
+	src.modules += new /obj/item/weapon/wrench/cyborg(src)
+	src.modules += new /obj/item/weapon/wirecutters/cyborg(src)
+	src.modules += new /obj/item/device/multitool/ai_detector(src)
+	src.modules += new /obj/item/weapon/extinguisher(src)
```

Comment:
Don't default borgs get an extinguisher?

Reply:
Mechoid told me they got the tiny one.

Reply:
Will double check, standby.

Reply:
Looks like the full sized on, from robot_modules.dm